src/components/elements/RecoverableError.tsx

```diff
@@ -4,20 +4,15 @@ import { useEffect } from 'react';
 import { sentryUser } from '@/modules/auth/api/sessions';
 import useAuth from '@/modules/auth/hooks/useAuth';
 
-const DEFAULT_THROW = import.meta.env.MODE !== 'production';
-
 interface Props {
   error: Error;
   blocking?: boolean;
 }
 
-const RecoverableError: React.FC<Props> = ({
-  error,
-  blocking = DEFAULT_THROW,
-}) => {
+const RecoverableError: React.FC<Props> = ({ error, blocking = false }) => {
   const { user } = useAuth();
   useEffect(() => {
-    if (blocking) {
+    if (blocking || import.meta.env.MODE !== 'production') {
```

Comment:
I didn't figure out why `const DEFAULT_THROW = import.meta.env.MODE !== 'production';` was causing problems with the storybook build. Something about the vite build and env variables. This is just a workaround.